recipe/meta.yaml

```diff
@@ -33,7 +37,10 @@ requirements:
     - boto3
     - fiona
     - gdal >=3.7
+    - geopandas
     - numpy
+    - osmium-tool
```

Comment:
🤔 It looks like `osmium-tool` was added to the `environment.yml` but *not* the `pyproject.toml` upstream:
https://github.com/ASFHyP3/asf-tools/compare/v0.6.0...v0.7.0#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711

Is this an actual dependency? If so, we should fix the `pyproject.toml`

Reply:
Yeah, we were talking about that. `osmium-tool` is available on conda-forge, but not on pypi

Reply:
`osmium-tool` is only in conda. It's a program rather than a python library. Would that still need to be in the pyproject.toml?

Reply:
Okay, then no, not in the `pyproject.toml`. But, that still means our PyPI distribution is broken as I don't see any handling to make that an optional dependency and notify people how to install it.

Reply:
I'll open an issue.

Reply:
Opened an issue here: https://github.com/ASFHyP3/asf-tools/issues/227

We can discuss it further there.